test/integration/migrate/index.js

```diff
@@ -268,12 +221,52 @@ module.exports = function(knex) {
 
   });
 
+  describe('knex.migrate.latest in parallel', function() {
+
+    afterEach(function() {
+      return knex.migrate.rollback({directory: 'test/integration/migrate/test'});
+    });
+
+    if (knex.client.dialect === 'postgres') {
+      it("is able to run two migrations in parallel in postgres", function () {
```

Comment:
Fair enough. Looks like only postgres and mssql are not doing implicit commits and oracle + mysql are https://massimotinelli.wordpress.com/2016/06/23/implicit-commit-ddl-oracle-vs-ms-sql-server/. Would be good to document that difference of functionality of different DB engines it is not obvious for many of the users.